web/firefoxcom.js

```diff
@@ -310,17 +310,67 @@ class FirefoxScripting {
 class MLManager {
   #enabled = new Map();
 
-  constructor({ enableAltText }) {
-    this.#enabled.set("altText", enableAltText);
+  constructor(options) {
+    this.eventBus = null;
```

Comment:
Nit: I'd just do the following instead.
```suggestion
  eventBus = null;
  
  constructor(options) {
```